frontend/ui/network/manager.lua

```diff
@@ -15,6 +15,39 @@ function NetworkMgr:readNWSettings()
     self.nw_settings = LuaSettings:open(DataStorage:getSettingsDir().."/network.lua")
 end
 
+-- Used after restoreWifiAsync() to make sure we eventually send a NetworkConnected event, as a few things rely on it (KOSync, c.f. #5109).
+function NetworkMgr:connectivityCheck(iter)
+    -- Give up after a while...
+    if iter > 4 then
```

Comment:
Not sure if the iter argument serves much purpose anymore?

Reply:
True ;).